scipy/optimize/optimize.py

```diff
@@ -311,9 +311,11 @@ def fmin(func, x0, args=(), xtol=1e-4, ftol=1e-4, maxiter=None, maxfun=None,
         Called after each iteration, as callback(xk), where xk is the
         current parameter vector.
     xtol : float, optional
-        Relative error in xopt acceptable for convergence.
+        Relative error in xopt acceptable for 
+        convergence(optimization halting).
```

Comment:
Not sure it's worth repeating that "convergence" means that a solution got found and optimization will halt. This is part of basic optimization "culture".